tests/unit/test-util/test.logger.js

```diff
@@ -122,12 +125,11 @@ describe('logger', () => {
       log.startCapturing();
       // $FLOW_IGNORE: fake process for testing reasons.
       log.write(packet({msg: 'message'}), {localProcess});
-      assert.equal(localProcess.stdout.write.called, false);
+      sinon.assert.notCalled(localProcess.stdout.write);
       // $FLOW_IGNORE: fake process for testing reasons.
       log.flushCapturedLogs({localProcess});
-      assert.equal(localProcess.stdout.write.called, true);
-      assert.equal(localProcess.stdout.write.firstCall.args[0],
-                   'message\n');
+      sinon.assert.calledWithMatch(
```

Comment:
This can be a specific value check so it should use `sinon.assert.calledWith()`. Try to avoid using `calledWithMatch()` unless you really want to ignore other arguments. As a general rule, always use `calledWith()` unless the first argument is an object.